shared/macros-oval.jinja

```diff
@@ -115,15 +118,28 @@
 
 {{#
   Macro to check if a parameter in a configuration file is set (Object definition).
-  This function can take five parameters:
+  This macro can take six parameters:
```

Comment:
Please change every occurrence of `This function` to `This macro` in the file.

Reply:
done